File: app/src/main/java/ir/touristland/Models/HotelPrices/Discount.java
package ir.touristland.Models.HotelPrices;

import com.google.gson.annotations.SerializedName;

public class Discount {

    @SerializedName("webservice")
    private String webservice;

    @SerializedName("sales")
    private String sales;

    public String getWebservice() {
        return webservice;
    }

    public void setWebservice(String webservice) {
        this.webservice = webservice;
    }

    public String getSales() {
        return sales;
    }

    public void setSales(String sales) {
        this.sales = sales;
    }

    @Override
    public String toString() {
        return
                "Discount{" +
                        "webservice = '" + webservice + '\'' +
                        ",sales = '" + sales + '\'' +
                        "}";
    }
}

File: app/src/main/java/ir/touristland/Models/HotelPrices/Discount.kt
package ir.touristland.Models.HotelPrices

import com.google.gson.annotations.SerializedName

class Discount {

    @SerializedName("webservice")
    var webservice: String? = null

    @SerializedName("sales")
    var sales: String? = null

    override fun toString(): String {
        return "Discount{" +
                "webservice = '" + webservice + '\''.toString() +
                ",sales = '" + sales + '\''.toString() +
                "}"
    }
}